README.md

```diff
@@ -195,27 +195,11 @@ For espresso developers we have written up a description of our workflow [here](
 
 # Extra Editor Configuration
 
-## Neovim
+Choose an async runtime to use before launching a text editor. This may be done by setting the environment RUSTFLAGS. For example:
 
-Copy `.nvim/example.env.lua` to `.nvim/env.lua`. The purpose of this file is to return to neovim the requisite feature flags.
-
-Then load this file when configuring `rust-tools`:
-
-```lua
-rust_tools.setup(
-  {server =
-    {settings =
-      {
-        ["rust-analyzer"] =
-          {cargo =
-            {features =
-              loadfile(".nvim/env.lua") and { loadfile(".nvim/env.lua")() } or "all"
-            }
-          }
-      }
-    }
-  }
-)
+```
+export RUSTFLAGS='--cfg async_executor_impl="tokio" --cfg async_channel_impl="tokio"'
```

Comment:
This will have unintended consequence of messing up `just` commands with the other executor ran from the same shell.
E.g. after
```sh
export RUSTFLAGS='--cfg async_executor_impl="tokio" --cfg async_channel_impl="tokio"'
nvim crates/hotshot/src/lib.rs
# enjoys rust-analyzer, exits nvim
just async_std_check # <-- fails, because just includes already-defined RUSTFLAGS
                     # when setting it's own and now we have executor set to both tokio and async_std
```

Reply:
Ah, good catch! I'll add a note about that to the readme.